bench-vortex/src/bin/clickbench.rs

```diff
@@ -89,15 +88,13 @@ struct DataFusionCtx {
 
 struct DuckDBCtx {
     duckdb_path: PathBuf,
-    tmp_dir: TempDir,
 }
 
 impl DuckDBCtx {
     pub fn duckdb_file(&self, format: Format) -> PathBuf {
-        self.tmp_dir
-            .path()
-            .to_path_buf()
-            .join(format!("hits-{format}.db"))
+        let dir = Path::new("bench-vortex/data/duckdb");
```

Comment:
@gatesn  Shouldn't the `dir` path relative to the Git root, not relative to the current working dir? For local runs, we can't assume they're the same.

Reply:
Yep, laziness on my part :(